src/components/Person/Person.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person as PersonType } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: PersonType,
+  findSlug: (name: string) => string,
+};
+
+export const Person: FC<Props> = ({
+  person,
+  findSlug,
+
+}) => {
+  const { peopleId } = useParams();
+
+  const motherName = person.motherName ? person.motherName : '-';
+  const fatherName = person.fatherName ? person.fatherName : '-';
+
+  const motherSlug = findSlug(motherName);
+  const fatherSlug = findSlug(fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': peopleId === person.slug })}
+    >
+      <td aria-label="person link">
+        <PersonLink
+          sex={person.sex}
+          name={person.name}
+          slug={person.slug}
+        />
```

Comment:
```suggestion
        <PersonLink person={person} />
```
You can just use whole person as a prop